src/gd_webp.c

```diff
@@ -291,7 +291,7 @@ BGD_DECLARE(void) gdImageWebpEx (gdImagePtr im, FILE * outFile, int quality)
 	if (out == NULL) {
 		return;
 	}
-	_gdImageWebpCtx(im, out, quality);
+	gdImageWebpCtx(im, out, quality);
```

Comment:
this is incorrect ... it should be using `_gdImageWebpCtx`

Reply:
I figured `gdImageWebpCtx()` was superior because it returns `void`, instead of returning an `int` that gets thrown away. Apparently not?

Reply:
`gdImageWebpCtx` is just going to call `_gdImageWebpCtx`, so it's only adding overhead to call `gdImageWebpCtx` ?

it's also about using the internal interfaces than having to go through the exported symbols which require a PLT entry

Reply:
got it!

Reply:
looks like these changes are still in here when they shouldn't be

Reply:
Sorry, this one dropped off the map. Reverted.